cfgov/core/tests/test_management_inactive_users.py

```diff
@@ -0,0 +1,110 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from StringIO import StringIO
+
+from datetime import timedelta
+
+import mock
```

Comment:
Nitpicky: conventional import order [seems to be to put this first](https://www.python.org/dev/peps/pep-0008/#imports).